kbase-extension/static/kbase/js/common/errorDisplay.js

```diff
@@ -52,7 +80,17 @@ define(['bluebird', 'uuid', 'common/html'], (Promise, Uuid, html) => {
                 {
                     class: `${cssBaseClass}__advice`,
                 },
-                [errorObject.advice]
+                ul(
+                    { class: `${cssBaseClass}__advice_list` },
+                    errorObject.advice.map((adv) => {
+                        return li(
+                            {
+                                class: `${cssBaseClass}__advice_list_item`,
+                            },
+                            adv
+                        );
+                    })
```

Comment:
moved this formatting code into this render function, rather than having it within the `convertInternalError` function